packages/ui/src/components/vuestic-components/va-date-picker/utils/child-props.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentOptionsBase, PropType, computed, ComputedRef, Prop } from 'vue'
+
+/**
+ * Accepts parent component props and return value only for child component props.
+ *
+ * Used to proxy child component props from parent.
+ */
+export const filterComponentProps = (propsValues: Record<string, any>, childProps: Record<string, any>): ComputedRef<Record<keyof typeof childProps, any>> => {
+  return computed(() => Object
```

Comment:
Maybe we could do it a bit faster :)

 ```
return Object.keys(childProps)
              .reduce((acc, propName) => {
                  acc[propName] = propsValues[propName]
                  return acc
              }, {})
}
```

Instead of:
```
return computed(() => Object
    .keys(childProps)
    .reduce((acc, propName) => {
      return { ...acc, [propName]: propsValues[propName] }
    }, {}),
  )
```

To not iterate object every time, if you would like.